assert/doc.go

```diff
@@ -1,5 +1,9 @@
 // Package assert provides a set of comprehensive testing tools for use with the normal Go testing system.
 //
+// # Note
+//
+// All functions in this package return a bool value indicating whether the test has passed.
```

Comment:
```suggestion
// All functions in this package return a bool value indicating whether the assertion has passed.
```

We shouldn't say "test" here because Go's testing package defines the test as the test function.